test/type_traits.cpp

```diff
@@ -631,6 +631,38 @@ TEST_CASE("add_pointer_t", "[meta.trans.ptr]") {
 
 // 23.15.7.6, other transformations
 
+// template <size_t Len, class... Types> struct aligned_union;
+TEST_CASE("aligned_union", "[meta.trans.other]") {
+  union U {
+    int x;
+    double y;
+  };
+
+  /* The member typedef type shall be a trivial standard-layout type suitable
+   for use as uninitialized storage for any object whose type is listed in
+   Types; its size shall be at least Len. */
+  {
+    using type = slb::aligned_union<0, int, double>::type;
+    CHECK(std::is_trivial<type>::value);
+    CHECK(std::is_standard_layout<type>::value);
+    CHECK(alignof(type) % alignof(U) == 0);
+
+    CHECK(sizeof(slb::aligned_union<sizeof(int) * 2, int>::type) >=
+          sizeof(int) * 2);
+  }
+
+  /* The static member alignment_value shall be an integral constant of type
+  size_t whose value is the strictest alignment of all types listed in Types.*/
```

Comment:
These comments are formatted weird :( probably because ClangFormat tries not to touch comments.

Reply:
How should they be formatted instead?

Reply:
I think for a paragraph like this maybe we should just do:
```
// The static member alignment_value shall be an integral constant of type
// size_t whose value is the strictest alignment of all types listed in Types.
{
  ...
}
```